scripts.js

```diff
@@ -28,11 +28,6 @@ var app = {
 
 		this._geolocateUserForAutocomplete();
 
-		this.addressInput.addEventListener('blur', function(e){
-			this.searchRepresentativesByAddress(e);
-			console.log('blur');
-		}.bind(this));
```

Comment:
This was causing issues because the blur event would fire before the autocomplete option was actually selected. This resulted in incorrect Congressmen being shown.